File: C++/2063. Vowels of All Substrings.cpp
class Solution {
public:
    long long countVowels(string w) {
        long long c=0;
        for(int i=0;i<w.size();i++){
            if(w[i]=='a'||w[i]=='e'||w[i]=='i'||w[i]=='o'||w[i]=='u')
            c+=(i+1)*(w.size()-i);
        }
        return c;
    }
};

File: JAVA/2063. Vowels of All Substrings.java
class Solution {
    public long countVowels(String w) {
        long c=0;
        for(int i=0;i<w.length();i++){
            if(w.charAt(i)=='a'||w.charAt(i)=='e'||w.charAt(i)=='i'||w.charAt(i)=='o'||w.charAt(i)=='u')
            c+=(long)(i+1)*(w.length()-i);
        }
        return c;
    }
}